discovery-provider/src/tasks/index_user_bank_backfill.py

```diff
@@ -397,14 +398,22 @@ def process_user_bank_txs(stop_sig: str):
                 USER_BANK_ADDRESS,
                 before=last_tx_signature,
                 limit=FETCH_TX_SIGNATURES_BATCH_SIZE,
+                retries=20,
             )
             transactions_array = transactions_history["result"]
             if not transactions_array:
                 intersection_found = True
                 logger.info(
-                    f"index_user_bank_backfill.py | No transactions found before {last_tx_signature}"
+                    f"index_user_bank_backfill.py | No transactions found before {last_tx_signature}, got {transactions_array}"
```

Comment:
Out of curiosity, was this (`transactions_array`) different for _real_ MIN_SIG vs non-real (error case)?

Reply:
I've been keeping an eye out but so far they've all been empty lists.